templates/macros/table_of_contents.html

```diff
@@ -50,5 +50,4 @@ <h3>{{ macros_translate::translate(key="table_of_contents", default="Table of Co
         {% endfor %}
     </ul>
 </div>
-
```

Comment:
❌ If we're not modifying the file, please don't change whitespace.

Reply:
I’m sorry for being careless.